packages/cosmwasm-stargate/src/signingcosmwasmclient.spec.ts

```diff
@@ -25,19 +25,19 @@ const { MsgDelegate } = codec.cosmos.staking.v1beta1;
 const { Tx } = codec.cosmos.tx.v1beta1;
 
 describe("SigningCosmWasmClient", () => {
-  describe("connectWithWallet", () => {
+  describe("connectWithSigner", () => {
     it("can be constructed", async () => {
       pendingWithoutWasmd();
       const wallet = await DirectSecp256k1HdWallet.fromMnemonic(alice.mnemonic, undefined, wasmd.prefix);
```

Comment:
Do we still like `Wallet` in these class names?

Reply:
The idea is this: signer is the interface. There are many ways to implement the signer. One way is having a wallet (term used for key storages here).